packages/toolkit/package.json

```diff
@@ -56,8 +56,9 @@
     "@phryneas/ts-version": "^1.0.2",
     "@size-limit/file": "^11.0.1",
     "@size-limit/webpack": "^11.0.1",
-    "@testing-library/react": "^13.3.0",
-    "@testing-library/user-event": "^13.1.5",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/react": "^16.0.1",
+    "@testing-library/user-event": "^14.5.2",
```

Comment:
Updated `@testing-library/react` to the latest version.
Changelogs for major updates: [v14.0.0](https://github.com/testing-library/react-testing-library/releases/tag/v14.0.0), [v15.0.0](https://github.com/testing-library/react-testing-library/releases/tag/v15.0.0), [v16.0.0](https://github.com/testing-library/react-testing-library/releases/tag/v16.0.0).

Since the version v16.0.0, `@testing-library/dom` was moved to a peer dependency, so we need to explicitly install it.

Updated `@testing-library/user-event` to the latest version. Major changes: [v14.0.0](https://github.com/testing-library/user-event/releases/tag/v14.0.0)